README.md

```diff
@@ -88,24 +88,7 @@ apply plugin: 'com.android.application'
 apply plugin: 'com.github.triplet.play'
 ```
 
-#### Snapshot builds
-
-If you're prepared to cut yourself on the bleeding edge of this plugin, snapshot builds are
-available through JitPack:
-
-```groovy
-buildscript {
-    repositories {
-        // ...
-        maven { url 'https://jitpack.io' }
-    }
-
-    dependencies {
-        // ...
-        implementation 'com.github.Triple-T:gradle-play-publisher:commitId' // E.g. 75bed587f8
-    }
-}
-```
+Snapshots of the development version are available in [Sonatype's `snapshots` repository](https://oss.sonatype.org/content/repositories/snapshots/).
```

Comment:
TBH, I don't think that we should explain this any further. Many open source projects just state that there are SNAPSHOT builds on Maven, and that's it. If someone is willing to be on the bleeding edge, it's likely that he already knows how to get the SNAPSHOT builds into their app.

@bhurling : Any opinion on this?

Reply:
That's fair, SGTM 👍

Reply:
I think it can really help us if people use snapshot releases. So I'd rather have them set this up as easy as possible. Let's add the regular Büro setup including maven url and or current snapshot version.

We no longer need to reference jitpack as that seems to be more of a way to test your own branch during plugin development

Reply:
@bhurling : Alright, I'll fix this up!